src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerAgreementOrchestrator.cs

```diff
@@ -26,19 +29,22 @@ public class EmployerAgreementOrchestrator : UserVerificationOrchestratorBase
         private readonly IMapper _mapper;
         private readonly IMediator _mediator;
         private readonly IReferenceDataService _referenceDataService;
+        private readonly IAccountRepository _accountRepository;
```

Comment:
Can't inject repo layer into orchestrator. Everything required should be passed from the mediator call.